docs/apps/datahub.md

```diff
@@ -11,10 +11,14 @@ The **Datahub** application offers a very intuitive and easy-to-use search inter
 - Powerful search experience using full-text search, advanced filters, sorting
 - Marking records as favorite to find them more easily later
 - Data visualization using maps, tables and charts
+  - Ability to customize basemap in map visualization with layers from 'xyz', 'wms' or 'wfs' service endpoint; or 'geojson'; or 'maplibre style' (see `map_layer` option in [configure guide](../guide/configure.md#map))
+  - Ability to configure a max feature count for WFS, with pagination, for an optimized display in table dataviz (see `max_feature_count` option in [configure guide](../guide/configure.md#map))
+- Improved record data fetching through cache-based requests, using OGC-client's built-in caching and records' update frequency awareness
```

Comment:
Maybe we could document the value that deactivates caching somewhere, I think it is "continual".

Reply:
Yes, it is 'continual' or more than one update per day. I have added a new "Cache" page.